<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>acceuil</class>
 <widget class="QFrame" name="acceuil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Open Sans&quot;;</string>
  </property>
  <widget class="QLabel" name="UserIcon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>285</y>
     <width>241</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Icons/user.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="Line">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>280</y>
     <width>20</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="CIN">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>280</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CIN :</string>
   </property>
  </widget>
  <widget class="QLabel" name="cinlabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>280</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%CIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="Nom">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>310</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom :</string>
   </property>
  </widget>
  <widget class="QLabel" name="nomlabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>310</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%Nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="Prenom">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prenom :</string>
   </property>
  </widget>
  <widget class="QLabel" name="prenomlabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%Prenom</string>
   </property>
  </widget>
  <widget class="QLabel" name="Date">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de naissance :</string>
   </property>
  </widget>
  <widget class="QLabel" name="datelabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>370</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="Nationalite">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nationalit√© :</string>
   </property>
  </widget>
  <widget class="QLabel" name="nationalitelabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%Nationalite</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bonus">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>430</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BonusLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bonus_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BonusLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bonus_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>490</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BonusLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bonus_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>520</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BonusLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bonuslabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BonusLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bonuslabel_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>460</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BonusLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bonuslabel_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>490</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BonusLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Bonuslabel_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>520</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BonusLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
